src/converters.js

```diff
@@ -215,6 +215,22 @@ function convertCsv(inCsv) {
   });
 }
 
+function convertJson(inJson) {
+  // TODO: provide more useful response to user if JSON.parse fails. Should tell them there was a problem
+  // with the JSON, and ideally, the location. Right now the app just "hangs" with preview loading animation.
+  return new Promise(function(resolve, reject) {
+    try {
+      let json = JSON.parse(inJson);
+      resolve({
+        fluxData: json
+      });
+    } catch (e) {
```

Comment:
I don't think you need try catch here. The exception will automatically cause the promise to reject.